---
date: "2016-04-27T00:00:00Z"
external_link: ""
# image:
#   caption: Photo by rawpixel on Unsplash
#   focal_point: Smart
# links:
# - icon: twitter
#   icon_pack: fab
#   name: Follow
#   url: https://twitter.com/georgecushen
# slides: example
summary: Using Docker and the Cloud
title: Projects in the cloud
url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""
---

I believe research should always be reproducible, even if that translates to extra work for the authors/developers. That line of thinking lead me to learn about `docker`. I have used it in the past to produce containers to freeze old versions of my package (e.g. see [docker image for scMerge](https://github.com/kevinwang09/scmerge_docker)). 

Some Google Cloud based workshops I have designed and administered are on the topics of

+ Single Cell RNA Analysis. Which was hosted at the [University of Sydney](https://github.com/SydneyBioX/SC_CPC_workshop), [Bioinformatics Summer 2019](https://github.com/SydneyBioX/BIS2019_SC) and [Cornell University](https://github.com/SydneyBioX/cornell_sc_workshop). An accompany wiki page is [here](https://github.com/SydneyBioX/SC_CPC_workshop/wiki). 

+ R, RStudio and Tidyverse. Which was hosted at [Bioinformatics Summer 2019](https://github.com/SydneyBioX/tidy2019). 

I also helped with the deployment of the following workshops at Bioinformatics Summer 2019:

+ [HiC analysis](https://github.com/ycao6928/testASMI)
+ [Phosphoproteomic data analysis](https://github.com/SydneyBioX/BIS2019_PhosR)

Every workshop that I hosted come with a GitHub Page website. All scripts for running the workshop (except passwords) are stored in their respective GitHub repo. 
